uv.lock

```diff
@@ -292,7 +292,7 @@ name = "click"
 version = "8.1.8"
 source = { registry = "https://pypi.org/simple" }
 dependencies = [
-    { name = "colorama", marker = "(platform_machine != 'aarch64' and platform_system == 'Windows' and sys_platform == 'linux') or (platform_system == 'Windows' and sys_platform != 'darwin' and sys_platform != 'linux')" },
+    { name = "colorama", marker = "sys_platform == 'win32'" },
```

Comment:
Why is this changing?

Reply:
@leseb I ran `uv sync` which updated it :(

Reply:
`uv` has been doing strange things to me locally too :(